include/logger/loggerConfig.h

```diff
@@ -260,9 +260,9 @@ typedef struct _PWMConfig{
 } PWMConfig;
 
 /// PWM frequency in Hz.
-#define MAX_PWM_CLOCK_FREQUENCY             200000
-#define MIN_PWM_CLOCK_FREQUENCY	 	10
-#define DEFAULT_PWM_CLOCK_FREQUENCY			10000
+#define MAX_PWM_CLOCK_FREQUENCY             65000
```

Comment:
A couple of things:
 
1) I don't see in the kernel coding guide a recommendation on this topic
2) if there is little risk to bugs, I would like to make this visual formatting an exception to the style guide. I really like seeing the visual formatting.